content/blog/multitenancy-for-kubernetes-clusters-using-hpe-alletra-6000-and-nimble-storage.md

```diff
@@ -0,0 +1,234 @@
+  csi.storage.k8s.io/node-stage-secret-namespace: hpe-storage
+  csi.storage.k8s.io/provisioner-secret-name: hpe-backend
+  csi.storage.k8s.io/provisioner-secret-namespace: hpe-storage
+  description: "Volume created by the HPE CSI Driver for Kubernetes"
+reclaimPolicy: Delete
+allowVolumeExpansion: true
+```
+
+At this point the Kubernetes cluster is ready to accept `PersistentStorageClaims`. Let's create one.
+
+```yaml
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: my-first-pvc
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 32Gi
+```
+
+Once the `PersistentVolume` has been bound we can inspect the array and determine that volume has been placed in the tenant's folder.
+
+![HPE Alletra 6000 Multitenancy for Kubernetes Clusters](/img/multitenancy-hpedev-screenshot.png "HPE Alletra 6000 Multitenancy for Kubernetes Clusters")
+
+That's it. Multitenant storage is now properly configured!
+
+# Container Storage Provider (CSP) access only
+
+Multitenant storage is only available today through the HPE Alletra 6000 and Nimble Storage Container Storage Provider for Kubernetes. The CSP uses an undisclosed REST API resource of the array to perform CRUD operations on objects that in turn are being tagged and grouped accordingly to create the notion of a full blown cloud experience both from the Kubernetes administrator perspective but most importantly for the storage administrator. The storage administrator does not need to worry about storage system credentials being compromised and wreaking havoc beyond the compartment they were assigned for.
+
+At the time of writing, no limitations on the CSP functionality is restricted by using a tenant instead of a system account with the "administrator" or "poweruser" role. HPE recommends switching over to the tenant model for Kubernetes clusters accessing HPE Alletra 6000 or Nimble Storage arrays running NimbleOS 6.0.0 or later.
+
+Visit [HPE Storage Container Orchestrator Documentation](https://scod.hpedev.io) (SCOD) to learn more about what storage resources are being exposed to tenants.
+
+# Example use cases
+
+There are plenty of different use cases that multitenancy enables for IT Ops looking to manage and secure storage resources for a diverse set of applications running on Kubernetes.
+
+## Ephemeral Inline Volumes
+
+End users that deploy applications on Kubernetes that require ephemeral storage at a capacity beyond of what a worker node is capable of providing, may use Ephemeral Inline Volumes. Before multitenancy, Kubernetes administrators had to share the `Secret` with the end user to allow provisioning of the Ephemeral Inline Volume. That is not very practical from a security standpoint as application administrators would have privileges on the storage array. Now, the end user may request a separate tenant to allow management of Ephemeral Inline Volumes securely for their application.
```

Comment:
Change "..capacity beyond of what a worker..." to "...capacity beyond what a worker" (drop the word "of")

Third sentence: add a comma after "...from a security standpoint"